stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/Mother.kt

```diff
@@ -30,6 +41,244 @@ import java.util.Date
 
 private val fixture = JFixture()
 
+internal fun randomChannelUpdatedEvent(
```

Comment:
I've added this one and `randomNewMessageEvent ` in my previous PR so we can remove them